tabular/src/autogluon/tabular/models/tab_transformer/TabModelBase.py

```diff
@@ -0,0 +1,48 @@
+from ...try_import import try_import_torch
+from .TabTransformerEncoder import EmbeddingInitializerClass
+
+class TabModelBaseClass:
+    try_import_torch()
```

Comment:
isn't this instantly evaluated upon file import? What is the point of this BaseClass?

Reply:
I don't think I need this one. The one I need is for TabNet which is inside tab_transformer_model.py. I'm cleaning up all of these super classes and the lazy imports now.